README.md

```diff
@@ -18,8 +18,6 @@ npm install @primer/components
 
 ## Usage
 
-**If you are upgrading from a version before `1.0.0-beta`, please read the [migration docs](migrating.md).**
```

Comment:
Since we're still in beta & we only had a few people who were using this before 1.0.0-beta, figured it would be better to remove this

Reply:
That's a good idea! We could keep those all in one file for safe keeping.

Another thought I had was that our release should ideally be our "migration docs". That's usually the first place I go in a repository if I need to find out what I need to update for a breaking change. I wonder if that would be the easiest way to maintain that information?